packages/errors/src/react/forbidden-error-page.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { HTMLAttributes } from 'react'
+
+import ErrorPage from './error-page'
+
+interface ForbiddenErrorPageProps extends HTMLAttributes<HTMLAnchorElement> {
+  href?: string
+}
+const ForbiddenErrorPage: React.FC<ForbiddenErrorPageProps> = ({
+  href = 'https://help.pluralsight.com/help/contact-us'
+}) => (
+  <ErrorPage
+    iconId="eyeball"
```

Comment:
Not sure it applies to this PR, but perhaps these should be enumed/exposed in vars so that they're code completable?

Reply:
Already typed.  We're good here.

iconId?: keyof typeof icons